std/string.d

```diff
@@ -62,19 +81,55 @@ strings, use $(D char[]), $(D wchar[]), or $(D dchar[]). The $(D xxxstring)
 types are preferable because they don't exhibit undesired aliasing, thus
 making code more robust.
 
-Not all functions for D _string handling are in this module. Functions related
-to Unicode and ASCII are found in $(LINK2 std_uni.html, std.uni) and
-$(LINK2 std_ascii.html, std.ascii), respectively. Other functions that have a
-wider generality than just strings can be found in std.algorithm and std.range.
-
-Functions
-$(XREF uni, icmp)
-$(XREF uni, toLower)
-$(XREF uni, toLowerInPlace)
-$(XREF uni, toUpper)
-$(XREF uni, toUpperInPlace)
-$(XREF format, format)
-are publicly imported.
+The following functions are publicly imported:
+
+$(BOOKTABLE ,
+$(TR $(TH Module) $(TH Functions) )
+    $(TR $(TD algorithm)
+        $(TD
+         $(XREF algorithm, cmp)
+         $(XREF algorithm, count)
+         $(XREF algorithm, endsWith)
+         $(XREF algorithm, startsWith)
+    ))
+    $(TR $(TD array)
+        $(TD
+         $(XREF array, join)
+         $(XREF array, split)
+    ))
+    $(TR $(TD format)
+        $(TD
+         $(XREF format, format)
+         $(XREF format, sformat)
+    ))
+    $(TR $(TD uni)
+        $(TD
+         $(XREF uni, icmp)
+         $(XREF uni, toLower)
+         $(XREF uni, toLowerInPlace)
+         $(XREF uni, toUpper)
+         $(XREF uni, toUpperInPlace)
+    ))
+)
+
+There is a rich set of functions for _string handling defined in other modules.
+Functions related to Unicode and ASCII are found in $(LINK2 std_uni.html, std.uni)
+and $(LINK2 std_ascii.html, std.ascii), respectively. Other functions that have a
+wider generality than just strings can be found in $(LINK2 std_algorithm.html,
+std.algorithm) and $(LINK2 std_range.html, std.range).
+
+See_Also:
```

Comment:
This isn't introduced by this PR, but for some reason, this section heading is showing up as `SeeAlso` (one word) instead of `See Also` (two words). Seems to be the artifact of some kind of postprocessing script in the dlang.org build. There are several places where such multi-word headings are used; we better fix them before next release...